inventory/group_vars/vagrant

```diff
@@ -13,8 +13,6 @@ elasticsearch_jvm_heap_size: 250m
 filebeat_output_logstash_hosts:
   - elk.vagrant:5044
 
-hoist_repo: /vagrant/.git
```

Comment:
This line also appears in `inventroy/group_vars/all` line 8:

```yaml
hoist_repo: https://github.com/BonnyCI/hoist.git
```

With `hoist_repo` removed from `bastion.yml`, I believe we no longer need that reference either?